contracts/protocol/integration/governance/SnapshotGovernanceAdapter.sol

```diff
@@ -0,0 +1,83 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache License, Version 2.0
+*/
+
+pragma solidity 0.6.10;
+pragma experimental "ABIEncoderV2";
+
+contract SnapshotGovernanceAdapter {
+
+
+    /* ============ State Variables ============ */
+
+    // delegate registry address
+    address public delegateRegistry;
+
+    /* ============ Constructor ============ */
+
+    /**
+     * Set state variables
+     *
+     * @param _delegateRegistry   delegate registry address
+     * 
+     */
+
```

Comment:
Don't add newlines between javadoc comments and the beginning of the function. Should look something like:
```
/*
 * Comment stuff
 * /
function functionName() public {

}
```